vcpkg.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "yomm2",
+  "version": "1.6.0",
+  "dependencies": [
+    "benchmark",
+    "boost-core",
+    "boost-dynamic-bitset",
+    "boost-mp11",
+    "boost-preprocessor",
+    "boost-test",
+    "vcpkg-tool-ninja"
```

Comment:
I have never used vcpkg in manifest mode, so I am not sure, but is `vcpkg-tool-ninja` needed (or it might be a host only dependency? https://learn.microsoft.com/en-us/vcpkg/concepts/manifest-mode#package-manifest-example)

Reply:
My goal is to make it as easy as possible for a developer, or CI, to grab everything that is needed to build the library, including test, examples and benchmarks. Yeah I considered moving the Ninja requirement to `CMakeUserPresets.json` but CI did not have it on Windows IIRC. Just being practical, it is a dev time only dep.